docs/pages/router/basics/layout.mdx

```diff
@@ -0,0 +1,166 @@
+
+```tsx _layout.tsx
+import { Stack } from 'expo-router';
+
+export default function StackLayout() {
+  return (
+    <Stack>
+      <Stack.Screen name="[productId]" options={{ headerShown: false }} />
+    </Stack>
+  );
+}
+```
+
+While it is possible to nest navigators, be sure to only do so when it is truly needed. In our example, if you wanted to then push **products/accessories/index.tsx** onto the stack, it's not necessary to have an additional **\_layout.tsx** in the **accessories** folder with a `Stack` navigator. That would define another stack inside the first one. It is fine to add folders that only affect the URL but otherwise use the same navigator as the parent folder.
+
+## Tabs
+
+Much like a stack, you can implement a tab navigator in your layout file, and all the routes directly inside that folder will be treated as tabs. Consider this file structure:
+
+<FileTree
+  files={[
+    ['app/(tabs)/\_layout.tsx'],
+    ['app/(tabs)/index.tsx'],
+    ['app/(tabs)/feed.tsx'],
+    ['app/(tabs)/profile.tsx'],
+  ]}
+/>
+
+In the **\_layout.tsx** file, return a `Tabs` component:
+
+```tsx _layout.tsx
+import { Tabs } from 'expo-router';
+import MaterialIcons from '@expo/vector-icons/MaterialIcons';
+
+export default function TabLayout() {
+  return (
+    <Tabs>
+      <Tabs.Screen
+        name="index"
+        options={{
+          title: 'Home',
+          tabBarIcon: ({ color }) => <MaterialIcons size={28} name="house.fill" color={color} />,
+        }}
+      />
+      <!-- Add more tabs here -->
+    </Tabs>
+  );
+}
+```
+
+This will cause the **index.tsx**, **feed.tsx**, and **profile.tsx** files to appear together in the same bottom tabs navigator. This `Tabs` component uses React Navigation's native bottom tabs and supports the same options.
+
+In the case of Tabs, you wll likely want to define the tabs in the navigator, as this will influence the order that tabs appear, the title, and the icon inside the the tab. The index route will be the default selected tab.
+
+## Slot
+
+In some cases, you may want a layout without a navigator. This is helpful for adding a header or footer around the current route, or for displayig a modal over any route inside a folder. In this case, you can use the `Slot` component, which serves as a placeholder for the current child route.
+
+Consider this file structure:
+
+<FileTree
+  files={[
+    ['app/social/\_layout.tsx'],
+    ['app/social/index.tsx'],
+    ['app/social/feed.tsx'],
+    ['app/social/profile.tsx'],
+  ]}
+/>
+
+Suppose you want to wrap any route inside the **social** folder with a header and footer, but you want navigating between the pages to simply replace the current page, rather than pushing a new page onto a stack, like what would happen with a `Stack` navigator. In the **\_layout.tsx** file, return a `Slot` component surrounded by your header and footer:
```

Comment:
```suggestion
For example, you may want to wrap any route inside the **social** directory with a header and footer, but you want navigating between the pages to simply replace the current page rather than pushing a new page onto a stack, like what would happen with a `Stack` navigator. In the **\_layout.tsx** file, return a `Slot` component surrounded by your header and footer:
```

What does "... like what would happen with a `Stack` navigator." means? It's a bit unclear in this context. Are you describing the behavior of the stack navigator.